includes/wikia/services/HubService.class.php

```diff
@@ -19,13 +20,13 @@ class HubService extends Service {
 	 *   WikiFactoryHub::CATEGORY_ID_LIFESTYLE or
 	 *   WikiFactoryHub::CATEGORY_ID_CORPORATE
 	 *
-	 * @param int $categoryId category id
+	 * @param integer $categoryId category id
 	 *
-	 * @return int
+	 * @return integer
```

Comment:
To unify if across at least this file. :wink:  http://www.phpdoc.org/docs/latest/references/phpdoc/types.html